element_deeplabcut/train.py

```diff
@@ -258,15 +245,16 @@ def make(self, key):
         }
         for k in ["shuffle", "trainingsetindex", "maxiters"]:
             train_network_kwargs[k] = int(train_network_kwargs[k])
+
         try:
             train_network(dlc_cfg_filepath, **train_network_kwargs)
         except KeyboardInterrupt:  # Instructions indicate to train until interrupt
-            pass
+            print("DLC training stopped via Keyboard Interrupt")
 
         snapshots = list(
             (
                 project_path
-                / get_model_folder(

```

Comment:
Shouldn't this remain as get_model_folder based on the latest deeplabcut version?